en/06_Project_Governance/05_Major_release_policy.md

```diff
@@ -0,0 +1,143 @@
+
+At launch, a Silverstripe CMS major release supports all PHP versions in full support. PHP versions in limited support are not supported at launch by new Silverstripe CMS major releases.
+
+Following the initial launch of a Silverstripe CMS major release, the development team aims to add forward compatibility for the next PHP release. e.g.: Silverstripe CMS 5 at launch will support PHP 8.1 and PHP 8.2. CMS 5 should receive official support for an eventual PHP 8.3 in early 2024.
+
+Support for end-of-life PHP releases is not dropped within a Silverstripe CMS major release line, unless it's necessary to address vulnerabilities or high impact bugs.
+
+Major release lines not in active development do not receive official support for later PHP releases.
+
+## Dependency management
+
+At launch, Silverstripe CMS major releases aim to have all PHP and JavaScript dependencies on their latest stable versions. For dependencies offering a *Long Term Support* version, the latest LTS of that dependency is preferred.
+
+A list of *fixed dependencies* for each Silverstripe CMS major release line is explicitly defined by the development team. *Fixed dependencies* do not change for the lifetime of a Silverstripe CMS major release line.
+
+Support for newer major releases of fixed dependencies is not added to Silverstripe CMS major release line, even if the currently used version is end-of-life. The only exception is if the upgrade is necessary to fix a high impact bug or security vulnerability. A Silverstripe CMS major release line only supports one major version of each fixed dependency.
+
+Non-fixed dependencies may be upgraded to new major releases or swapped out altogether within a Silverstripe CMS release line.
+
+### Symfony support commitments
+
+Symfony is a collection of common PHP libraries Silverstripe CMS uses. At launch, Silverstripe CMS major releases track the latest stable major version of Symfony dependencies. A Silverstripe CMS major release line does not add support for later Symfony major versions.
+
+## Commercially supported module
+
+Commercially supported modules are reviewed with each new major release. At launch, the list of commercially supported modules for that release line is explicitly defined.
+
+New modules may be added to the commercially supported modules list during a Silverstripe CMS major release line's lifespan. Modules are not withdrawn from the commercially supported modules list for a Silverstripe CMS major releases line unless external factors make it impractical to continue support. e.g. The module requires a discontinued third party service to work as intended.
+
+Modules that are supported for one major release line may not be supported for follow up release lines.
+
+## Expected Silverstripe CMS major release timeline
+
+This timeline is provided as an indication only.
+
+![Timeline of planned Silverstripe CMS major releases](../../_images/major_release_timeline.jpg)
```

Comment:
Now that there is a graphic I am even more certain that this belongs on the roadmap page rather than here, since it is talking about _specific_ releases (and will therefore eventually be outdated), where this policy is intended to be release-agnostic - it doesn't care about and _specific_ releases.

[Your reasoning last time](https://github.com/silverstripe/developer-docs/pull/54#discussion_r939484942) I mentioned the roadmap page for this was:
> That page will probably only show current releases ... not future ones.

But a roadmap is really meant to be about the future, so I think that page is a perfect fit for a graphic that explicitly talks about the intended release schedule of specific future releases.

I don't think we need a graphic for this policy, but if we do have one I think it should not refer to any specific releases. This whole section in fact probably just doesn't really belong here - it should be on the roadmap page where people will be explicitly looking for release timeline info.

Reply:
All that said, I don't feel super strongly about this - so if you want to keep the graphic here I'm okay with it in the interest of getting this over the line.

Reply:
The main value of the graphic is that it visually conveys what releases will happen one and the overlap between releases. 

Happy to move the graphic to an eventual roadmap page once SSORG is rebuilt. In the short term, I think it provides value.